packages/common/string/src/filter.ts

```diff
@@ -0,0 +1,17 @@
+import { hasNoItems } from '@angular-ru/common/array';
+import { isFunctionLike } from '@angular-ru/common/function';
+
+type FilterPredicateFn = (char: string) => boolean;
+
+export function filter(value: string, predicate: string[] | FilterPredicateFn = []): string {
```

Comment:
`string[] | RegExp[]` what do you think about improved it?

Reply:
good idea
I added tests
```
        it('should use RegExp for filtering text', () => {
            expect(filter('aaabbbccc', /a/)).toEqual('a');
            expect(filter('aaabbbccc', /a/g)).toEqual('aaa');
            expect(filter('aaabbbccc', /[a,b]+/)).toEqual('aaabbb');
        });
```